java/src/main/java/ai/rapids/cudf/ColumnVector.java

```diff
@@ -2056,7 +2082,22 @@ private static native long substringColumn(long columnView, long startColumn, lo
    */
   private static native long stringEndWith(long cudfViewHandle, long compString) throws CudfException;
 
-  private static native long matchesRe(long cudfViewHandle, String compString) throws CudfException;
+  /**
+   * Native method for checking if strings match the passed in regex pattern from the
+   * beginning of the string.
+   * @param cudfViewHandle native handle of the cudf::column_view being operated on.
+   * @param pattern string regex pattern.
+   * @return native handle of the resulting cudf column containing the boolean results.
+   */
+  private static native long matchesRe(long cudfViewHandle, String pattern) throws CudfException;
+
+  /**
+   * Native method for checking if strings match the passed in regex pattern.
```

Comment:
is this matches or contains?

Reply:
contains will check for the regex anywhere in the string. the only difference between the two docs is `from the
   beginning of the string.` in the case of matches. I am following the cudf doc so maybe I can be more clear in the next version.

Reply:
Re-wording done.